applications/RANSApplication/tests/incompressible_potential_flow_solver_formulation_tests.py

```diff
@@ -0,0 +1,29 @@
+import KratosMultiphysics as km
+import KratosMultiphysics.KratosUnittest as UnitTest
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+from KratosMultiphysics.RANSApplication.test_utilities import RunParametericTestCase
+
+
+class IncompressiblePotentialFlowSolverFormulationTest(UnitTest.TestCase):
+    def setUp(self):
+        # Set to true to get post-process files for the test
+        self.print_output = False
+
+    def testSteady(self):
+        self._runTest()
+
+    def _runTest(self):
+        self.addCleanup(lambda: kratos_utilities.DeleteTimeFiles("BackwardFacingStepTest"))
+
+        self.parameters = {}
+        if (km.IsDistributedRun()):
+            self.parameters["<PARALLEL_TYPE>"] = "MPI"
+        else:
+            self.parameters["<PARALLEL_TYPE>"] = "OpenMP"
+        RunParametericTestCase(
```

Comment:
```suggestion
        RunParametricTestCase(
```

Reply:
Agreed, and changed locally. Will push once you finish review :)